### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

**Review by kiritofeng — APPROVED** [review]
LGTM

**Comment by kiritofeng** [comment]
ok to test

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 =====
 
-Python [AGPLv3](LICENSE) contest judge backend for the [DMOJ site](http://github.com/DMOJ/online-judge) interface. See it in action at [dmoj.ca](https://dmoj.ca/)!
+Python [AGPLv3](LICENSE) contest judge backend for the [DMOJ site](https://github.com/DMOJ/online-judge) interface. See it in action at [dmoj.ca](https://dmoj.ca/)!
 
 <table>
 <tr>
 <td>
-<a href="http://dmoj.ca">
+<a href="https://dmoj.ca/">
```
Comment (fluix-dev) [inline README.md:9]: Everything else seems fine, but why the addition of the trailing slash on all these?

Comment (int-y1) [reply README.md:9]: I entered `dmoj.ca` into my browser and copied `https://dmoj.ca/`.

**Review by Xyene — APPROVED** [review]
LGTM, thanks

**Pull request merged**